specification/network/resource-manager/Microsoft.Network/preview/2021-02-01-preview/examples/NetworkManagerAdminRuleCollectionDelete.json

```diff
@@ -0,0 +1,14 @@
+{
+  "parameters": {
+    "api-version": "2021-02-01-preview",
+    "subscriptionId": "subId",
+    "resourceGroupName": "rg1",
+    "networkManagerName": "testNetworkManager",
+    "configurationName": "myTestSecurityConfig",
```

Comment:
Should it be myTestAdminConfig

Reply:
Same Q for all files

Reply:
They are SecurityAdminConfig and SecurityUserConfig, and they share the same data model secuirtymodel, so it is not wrong to call it myTestSecurityConfig here